backend/app/__init__.py

```diff
@@ -83,3 +83,88 @@ def one_item(item_id):
 @app.route("/")
 def index():
     return "Works :)", 200
+
+#DB for the people opinions/posts
+@app.route("/opinion", methods=["POST", "GET"])
+def opinions_list():
+    if request.method == "POST":
+        body = request.json
+        product_name = body["productName"]
+        product_opinion = body["productOpinion"]
+        product_link = body["poductLink"]
```

Comment:
shouldn't product_link be product_id instead?

Reply:
Nono but I can add that too, supposly people has to load the link where they got their product

Reply:
Ohh right! Totally forgot about that